agri-store-react/src/Pages/CartPage.js

```diff
@@ -4,12 +4,15 @@ import { Header } from "../Components/Header/Header";
 import { Footer } from "../Components/Footer/Footer";
 import { useCartContext } from "../Components/Context/CartPageContext";
```

Comment:
you can export all context in index.js file inside context folder & import all context from that file only